### ISSUE TITLE
Second light source jumps to a new position when a drag is started.

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/geometric-optics#191]
When a drag is started on the second light source, it immediately jumps to a different position.

Steps to reproduce:

1. Run the sim
2. Got to Lens screen
3. Select lamp from the source combo box
4. Check the "Second Source" checkbox
5. Press on the red lamp. It will jump to the left and slightly up.

**Comment by pixelzoom** [comment]
In #82 (duplicate), @veillette said:
> As part of the work on #79, we noticed that there is an offset to the second source on start event off the dragListener. This is present solely on the movable source of light and  not on the movable object.

**Comment by pixelzoom** [comment]
In #120, I discovered that the touchArea was incorrect for the light source -- it was still set to the second source point.  See screenshot below.  

I fixed that in https://github.com/phetsims/geometric-optics/commit/9657052641de780632da7aeb5fb343b16a58d570, in SecondSourceNode.js:

```js
        this.touchArea = secondSourceImage.localBounds.dilateXY( 5, 5 );
```

Unfortunately, that did not fix this issue.

<img width="999" alt="screenshot_1257" src="https://user-images.githubusercontent.com/3046552/133696944-2846f2c6-4f4d-4351-ad21-fa7d5a5f744c.png">

**Comment by pixelzoom** [comment]
The position of the second light source is currently being adjusted (incorrectly) in SecondSourceNode.js, by subtracting `LIGHT_SOURCE_OFFSET`.   This should be handled in SourceObject.js, as it is for the first light source, by using `Representation.LIGHT.rightFacingUprightOffset` to adjust the positon of the second light source.

**Comment by pixelzoom** [comment]
Adding this to the Prototype milestone, since it's a relatively serious UI problem.

**Comment by pixelzoom** [comment]
The above commit fixes LIGHT_SOURCE_OFFSET, so that the rays now appear to come out of the light at the correct spot. No idea why it was so wrong.

**Comment by pixelzoom** [comment]
The "jumps" problem is fixed in https://github.com/phetsims/geometric-optics/commit/c4440176497407ffb16926c4c349f81f9135aa9c.  I solved the problem by adding option `offsetPosition` to the DragListener. But I don't understand the specific value that I found to be necessary - it's a function of `LIGHT_SOURCE_OFFSET`.   I've added a TODO in the code to circle back after prototype publication.  And all of this should really be handled in the model, not the view.

I'll go ahead and close this issue, since this is acceptable for the prototype.
